File: VB/GridWithExpressions/Task.vb
Namespace GridWithExpressions

    Public Class Task

        Public Sub New(ByVal name As String, ByVal start As Date, ByVal finish As Date, ByVal isComplete As Boolean)
            Me.Name = name
            StartDate = start
            FinishDate = finish
            IsCompleted = isComplete
        End Sub

        Public Property Name As String

        Public Property FinishDate As Date

        Public Property StartDate As Date

        Public Property IsCompleted As Boolean
    End Class
End Namespace

File: CS/GridWithExpressions/Task.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace GridWithExpressions {
    public class Task {

        public Task(string name, DateTime start, DateTime finish, bool isComplete) {
            Name = name;
            StartDate = start;
            FinishDate = finish;
            IsCompleted = isComplete;
        }

        public string Name { get; set; }
        public DateTime FinishDate { get; set; }
        public DateTime StartDate { get; set; }
        public bool IsCompleted { get; set; }
    }
}
